infra/gcp/ensure-staging-storage.sh

```diff
@@ -44,6 +44,7 @@ STAGING_PROJECTS=(
     cip-test
     cluster-api
     cluster-api-aws
+    cluster-api-openstack
```

Comment:
Follow up from separate conversation, I think `capi-openstack` is probably better here, and we can look at porting over renaming `cluster-api-aws` to `capi-aws` as part of v1alpha3 work.

Reply:
I shortened the name as discussed